cmd/cert-checker/main.go

```diff
@@ -386,10 +384,15 @@ func main() {
 	cmd.FailOnError(err, "Couldn't load DB URL")
 	saDbMap, err := sa.NewDbMap(saDbURL, config.CertChecker.DBConfig.MaxDBConns)
 	cmd.FailOnError(err, "Could not connect to database")
-	scope := metrics.NewPromScope(prometheus.DefaultRegisterer)
 
-	// Collect and periodically report DB metrics using the DBMap and prometheus scope.
-	sa.InitDBMetrics(saDbMap, scope)
+	// Collect and periodically report DB metrics
```

Comment:
Totally unrelated to your PR, but since I changed the `InitDBMetrics` impl to use a custom collector the "periodically report DB metrics" part isn't really true. It happens based on the prometheus scrape interval.

Can you fix this comment while you're diff'ing this file? I think the comment isn't very useful anymore and can probably be rm'd.

```suggestion
```